resources/add_to_list.rb

```diff
@@ -25,7 +25,7 @@
 
     regexdelim = []
     new_resource.delim.each do |delim|
-      regexdelim << escape_regex(delim)
+      regexdelim << escape_string(delim)
```

Comment:
Would this work here: string = Regexp.escape(delim)
The we could get rid of the escape string methods.